Dockerfile

```diff
@@ -1 +1,14 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/hello-world:latest
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:20210701-2f4a4c7
```

Comment:
Do we want to use `latest` somehow, or always refer to a specific version of the base image?  If we refer to a specific version of `ark_base`, then every time `ark_base` changes we have rolling changes to the other repositories ... which might be OK I guess.

Reply:
We have added a latest tag to our builds now, please update this line to use it.

Reply:
updated to latest